model/mod_list.py

```diff
@@ -134,16 +251,18 @@ def handle_rows_inserted(self, parent: QModelIndex, first: int, last: int) -> No
                 widget = ModListItemInner(
                     data, self.width(), self.steam_icon_path, self.ludeon_icon_path
                 )
+                if data.get("invalid"):
+                    widget.main_label.setStyleSheet("QLabel { color : red; }")
                 item.setSizeHint(widget.sizeHint())
                 self.setItemWidget(item, widget)
-                self.package_ids.add(data["packageId"])
-                self.item_added_signal.emit(data["packageId"])
+                self.uuids.add(data["uuid"])
+                self.item_added_signal.emit(data["uuid"])
 
-        if len(self.package_ids) == self.count():
+        if len(self.uuids) == self.count():
             self.list_update_signal.emit(str(self.count()))
 
     def handle_other_list_row_added(self, package_id: str) -> None:
```

Comment:
This is no longer package_id, right? The function def needs to be fixed.

Reply:
Fixed